wolfcrypt/src/dsa.c

```diff
@@ -173,7 +173,7 @@ int wc_MakeDsaKey(WC_RNG *rng, DsaKey *dsa)
         return MEMORY_E;
     }
 
-    SAVE_VECTOR_REGISTERS();
+    SAVE_VECTOR_REGISTERS(;);
```

Comment:
This should not be necessary, because compliant C preprocessors always pass an empty argument when invoked with `()` as the arg list.  Did you run across a compiler/preprocessor that doesn't handle this correctly?

Reply:
Agreed, however see about halfway though the list, above. Windows compiler complained:

```
dsa.c(176,5): warning C4003: not enough arguments for function-like macro invocation 'SAVE_VECTOR_REGISTERS'
```

Reply:
This change is fine. `SAVE_VECTOR_REGISTERS` expects one arg.